API/testrunner/devices/artik530.py

```diff
@@ -111,7 +111,8 @@ def execute(self, testset, test):
             command += ' --no-memstat'
 
         if self.env['info']['coverage'] and self.app == 'iotjs':
-            command += ' --coverage-port %s' % utils.read_port_from_url(self.env['info']['coverage'])
+            command += ' --coverage-port %s' %\
+                            utils.read_port_from_url(self.env['info']['coverage'])
```

Comment:
The indentation is wrong and maybe you could organize out the result of the utils.read_port_from_url function into a port variable instead of using this backslash character.